jest.config.js

```diff
@@ -1,11 +1,12 @@
-const E2E_MODE = process.env.npm_lifecycle_event === 'e2e';
+const E2E_MODE = process.env.npm_lifecycle_event === 'e2e' || 'ci';
```

Comment:
I don't think there are any unintended side effects in the current codebase, but this results in the value being `E2E_MODE=true` for `npm run e2e` and `E2E_MODE='ci'` for all other npm commands.

The reason that doesn't disrupt anything is because `'ci'` is indeed a truthy value, and we're not using Jest for non-puppeteer stuff.

```suggestion
const E2E_MODE = ['e2e', 'ci'].includes(process.env.npm_lifecycle_event);
```

You can test the code suggestion by adding `console.log('E2E_MODE', E2E_MODE);` on the next line after it:

```sh
$ npm run e2e

> common-design@4.0.5 e2e
> jest _tests/*.test.js

E2E_MODE true
^C

$ npm run ci

> common-design@4.0.5 ci
> jest --maxWorkers=2 _tests/*.test.js

E2E_MODE true
^C
```